SAMtemplates/load_balancer/main.yaml

```diff
@@ -0,0 +1,146 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: |
+  Main template for Load Balancer with all resources included, supporting mutual TLS
+
+Parameters:
+  StackName:
+    Type: String
+
+  EnableMutualTLS:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+
+  TruststoreVersion:
+    Type: String
+
+  TruststoreFile:
+    Type: String
+
+  VPC:
+    Type: AWS::EC2::VPC::Id
```

Comment:
The VPC & subnet info can just be directly imported as mentioned in other comments